packages/legend-extension-external-language-morphir/src/MorphirGenerationExtension.ts

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
Let's follow our convention in other extension packages to prefix file name with `EL_` to denote `external language`

Reply:
Fixed.